package.json

```diff
@@ -57,9 +57,13 @@
     "memfs": "^3.1.2",
     "prettier": "^2.0.2",
     "ts-jest": "^25.3.0",
+    "ts-node": "^8.8.2",
     "typescript": "^3.8.3",
     "wait-for-expect": "^3.0.2"
   },
+  "peerDependencies": {
+    "ts-node": "^8.8.2"
+  },
```

Comment:
Does this cause a warning if they don't install `ts-node`?

Reply:
Yeah, it will display a warning. tbh I was kinda on the fence on whether to add `ts-node` to peer deps knowning that it was a peer dep, but unfortunately there's no concept of optional peer dependencies that do not cause a warning to be logged. 

I settled on that idea that it was probably better to be explicit about peers, but I have no strong opinions on this.

Reply:
I feel like warnings, when not resolved, are "broken windows", inviting vandalism at worst, or confusing people who wonder why they're being warned at the very beginning of their experience.

I would be inclined to handle this by not having the peer dependency - since it is optional - and simply have the code that requires `ts-node` handle the require error, providing a nice output saying "It looks like you're building with TypeScript. Please add `ts-node` as a dependency of your project to allow the CLI to compile your code." (or something like that). We would document the need as well, of course, in the bit that says a project can be done as TS or not.

That said, it's nothing that needs to be resolved right now. Also, I'd be curious how the community does optional dependencies in general.

Reply:
kk, I'll remove the peer dep in one of the next prs prior to the next publish

Reply:
Actually, looks like I created a conflict, removing now